crypto/vrf/vrf_test.go

```diff
@@ -29,23 +30,28 @@ func TestHonestComplete(t *testing.T) {
 	}
 }
 
-func TestConvertSecretKeyToPublicKey(t *testing.T) {
-	pk, sk, err := GenerateKey(nil)
+func TestConvertPrivateKeyToPublicKey(t *testing.T) {
```

Comment:
Do you think it doesn't make sense to test if `Public()` succeeds and the returned bytes match `sk[32:]`? I would prefer to have a test for `Public` and this were the only two checks that came to my mind.